src/networks/standalone_sgan/fid_score.py

```diff
@@ -45,7 +48,11 @@ def __init__(self, imgs_original, batch_size=64, dims=2048, n_samples=10000, cud
         self.n_samples = n_samples
         self.cuda = cuda
         self.verbose = verbose
-        self.dims = 10    # For MNIST the dimension of feature map is 10
+        # For MNIST
+        # self.dims = 10
+
+        # For CIFAR
+        self.dims = 2048
```

Comment:
Could you add a new input parameter to select the data set? 
Thus, if self.dataset=='MNIST' or self.dataset=='CIFAR' you change dims, the way of loading the model, etc.